packages/calcite-components/src/components/panel/panel.stories.ts

```diff
@@ -440,3 +440,24 @@ export const withNoHeaderBorderBlockEnd_TestOnly = (): string =>
   html`<calcite-panel style="--calcite-panel-header-border-block-end:none;" height-scale="s" heading="My Panel"
     >Slotted content!</calcite-panel
   >`;
+
+export const contentTopSlot_TestOnly = (): string => html`
+  <div style="height: 300px; width: 400px; display: flex">
```

Comment:
Is this outer element needed for the `content-top` slot screenshot?

Reply:
The outer element is there to restrict the height of the panel so we can see how scrolling of the slotted content affects the header (it should be sticky).